### PR TITLE
Simplify long strings where possible in GitHub CI `podman_runner`.

### PR DESCRIPTION
**Pull request by szachovy**
## What does this PR change?

Having names like `uyuni-server-all-in-one-test` and `test-all-in-one` makes long commands (especially these with `exec`) harder to read, and it is unnecessary.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/24071
Port(s): N/A

- [x] **DONE**

## Changelogs

- [x] No changelog needed

### CODE REVIEW

**Review by srbarrios — APPROVED** [review]

File: testsuite/podman_runner/07_start_server.sh [hunk 1]
```diff
@@ -19,14 +19,14 @@ sudo -i podman run --cap-add AUDIT_CONTROL --rm \
     -v ${src_dir}/spacewalk/setup/share/tomcat_java_opts_suse.conf:/etc/tomcat/conf.d/tomcat_java_opts_suse.conf \
     -v ${src_dir}/java/conf/default/rhn_taskomatic_daemon.conf:/usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf \
     -v /sys/fs/cgroup:/sys/fs/cgroup:rw \
-    -v /tmp/test-all-in-one:/tmp \
+    -v /tmp/testing:/tmp \
```
Comment (srbarrios) [inline testsuite/podman_runner/07_start_server.sh:22]: Personally, I would just do -v /tmp:/tmp
But matter of taste.

Comment (szachovy) [reply testsuite/podman_runner/07_start_server.sh:22]: Is there a possibility to overwrite some content of these files between intentional and non-intentional changes?
```
drwxrwxrwt 14 root     root     12288 Apr 15 13:30 .
drwxr-xr-x 21 root     root      4096 Apr 15 13:16 ..
drwxrwxrwt  2 root     root      4096 Apr 15 13:16 .ICE-unix
drwxrwxrwt  2 root     root      4096 Apr 15 13:16 .Test-unix
drwxrwxrwt  2 root     root      4096 Apr 15 13:16 .X11-unix
drwxrwxrwt  2 root     root      4096 Apr 15 13:16 .XIM-unix
drwxrwxrwt  2 root     root      4096 Apr 15 13:16 .font-unix
prwx------  1 runner   docker       0 Apr 15 13:17 clr-debug-pipe-1589-68[20](https://github.com/uyuni-project/uyuni/actions/runs/8689836011/job/23828379459#step:25:21)-in
prwx------  1 runner   docker       0 Apr 15 13:17 clr-debug-pipe-1589-6820-out
prwx------  1 runner   docker       0 Apr 15 13:17 clr-debug-pipe-1607-7103-in
prwx------  1 runner   docker       0 Apr 15 13:17 clr-debug-pipe-1607-7103-out
prwx------  1 root     root         0 Apr 15 13:16 clr-debug-pipe-591-893-in
prwx------  1 root     root         0 Apr 15 13:16 clr-debug-pipe-591-893-out
srw-------  1 runner   docker       0 Apr 15 13:17 dotnet-diagnostic-1589-6820-socket
srw-------  1 runner   docker       0 Apr 15 13:17 dotnet-diagnostic-1607-7103-socket
srw-------  1 root     root         0 Apr 15 13:16 dotnet-diagnostic-591-893-socket
-rw-r--r--  1 runner   docker      26 Apr 15 13:29 opensuse_product_uuid
-rw-r--r--  1 runner   docker      23 Apr 15 13:25 opensuse_ssh_product_uuid
-rw-r--r--  1 runner   docker      20 Apr 15 13:29 rh_product_uuid
drwx------  3 root     root      4096 Apr 15 13:16 snap-private-tmp
drwx------  3 root     root      4096 Apr 15 13:16 systemd-private-bc805fe07794443abbaa8b5ff2e1b8ba-chrony.service-QKCVdD
drwx------  3 root     root      4096 Apr 15 13:16 systemd-private-bc805fe07794443abbaa8b5ff2e1b8ba-haveged.service-3fXsy7
drwx------  3 root     root      4096 Apr 15 13:16 systemd-private-bc805fe07794443abbaa8b5ff2e1b8ba-systemd-logind.service-QfwPUf
drwx------  3 root     root      4096 Apr 15 13:16 systemd-private-bc805fe07794443abbaa8b5ff2e1b8ba-systemd-resolved.service-1vzw5E
drwxrwxrwt 20 root     root     1[22](https://github.com/uyuni-project/uyuni/actions/runs/8689836011/job/23828379459#step:25:23)88 Apr 15 13:29 testing
-rw-r--r--  1 runner   docker      24 Apr 15 13:30 ubuntu_product_uuid
drwxr-xr-x  3 www-data www-data  4096 Apr 15 13:16 www-data-temp-aspnet-0
```

I agree that simply doing `-v /tmp:/tmp` would omit more accidental errors. However I don't know if it's good to mount all this crap to containers.

I am open to both solutions, if no strong opinion appear, I will leave it as it is.

Comment (srbarrios) [reply testsuite/podman_runner/07_start_server.sh:22]: Leave it as it is, I guess is safer.

**Comment by szachovy** [comment]
Names of our registry images could also be simplified to provide a more direct definition of what lies behind them.

Screenshot from 2024-04-16 15-42-40


I decided not to interfere with this because it could be a disruptive change, resulting in having 2x as many registry images for some time, especially when the content within them would be the same.

**Comment by srbarrios** [comment]
> Names of our registry images could also be simplified to provide a more direct definition of what lies behind them.
> 
> Screenshot from 2024-04-16 15-42-40
> 
> 
> I decided not to interfere with this because it could be a disruptive change, resulting in having 2x as many registry images for some time, especially when the content within them would be the same.
> 

Yup. Make sense to work on that in a different PR.

**Pull request merged**